features/bevragen/persoon/verblijfplaats/dev/fields-gba.feature

```diff
@@ -280,3 +280,85 @@ Rule: wanneer één of meerdere velden van een verblijfplaats buitenland wordt g
     | verblijfplaatsBinnenland.verblijfadres                                              |
     | verblijfplaatsBinnenland.datumVan,verblijfplaatsBinnenland.verblijfadres.huisnummer |
     | verblijfplaatsBinnenland.verblijfadres.regel1                                       |
+
+Rule: als één of meerdere (gegevensgroep) velden van een verblijfplaats wordt gevraagd en de verblijfplaats heeft in onderzoek gegevens, dan worden deze ook geleverd
```

Comment:
Uit de onderstaande scenario's blijkt niet wat er geleverd wordt wanneer aanduiding in onderzoek is iets dat niet gevraagd wordt of zelfs niet in de resource zit. Bijvoorbeeld 080910 (zit op niveau Persoon), 081010 (niet gevraagd), 081020 (niet in resource), 081410 (zit in immigratie)

Reply:
> Uit de onderstaande scenario's blijkt niet wat er geleverd wordt wanneer aanduiding in onderzoek is iets dat niet gevraagd wordt of zelfs niet in de resource zit. Bijvoorbeeld 080910 (zit op niveau Persoon), 081010 (niet gevraagd), 081020 (niet in resource), 081410 (zit in immigratie)

081010 (niet gevraagd) wordt toch geïllustreerd met de voorbeelden op regel 311? datumVan wordt gevraagd en huisletter is in onderzoek